Sources/Operation.swift

```diff
@@ -116,21 +133,24 @@ extension OperationBuilder: Builder {
         let responses = try self.responses.mapValues { response in
             try ResponseBuilder.resolve(swagger, reference: response)
         }
-
-        let defaultResponse = try self.defaultResponse.map { response in
-            try ResponseBuilder.resolve(swagger, reference: response)
+        let callbacks = try self.callbacks.mapValues {
+            try CallbackBuilder.resolve(swagger, reference: $0)
+        }
+        var requestBody: Either<RequestBody, Structure<RequestBody>>?
+        if let requestBodyBuilder = self.requestBody {
+            requestBody = try RequestBodyBuilder.resolve(swagger, reference: requestBodyBuilder)
         }
 
-        return Operation(
-            tags: self.tags,
-            summary: self.summary,
-            description: self.description,
-            externalDocumentation: externalDocumentation,
-            parameters: parameters,
-            responses: responses,
-            defaultResponse: defaultResponse,
-            deprecated: self.deprecated,
-            identifier: self.identifier,
-            security: self.security)
+        return Operation(tags: self.tags,
+                         summary: self.summary,
+                         description: self.description,
+                         externalDocumentation: externalDocumentation,
+                         identifier: self.identifier,
+                         parameters: parameters,
+                         requestBody: requestBody,
+                         responses: responses,
+                         callbacks: callbacks,
+                         deprecated: self.deprecated,
+                         security: self.security)
     }
 }
```

Comment:
### [Operation Object](https://swagger.io/specification/#operationObject)

This object seems to differ much in OAS2 and OAS3 - possibly we would need 2 different 'versioned' objects.